lte/gateway/python/magma/mobilityd/ip_descriptor_map.py

```diff
@@ -50,24 +50,14 @@
 class IpDescriptorMap:
 
     def __init__(self,
-                 persist_to_redis: bool = True,
-                 redis_port: int = 6379):
+                 ip_states: MutableMapping[str, MutableMapping[str, IPDesc]]):
         """
-
         Args:
-            persist_to_redis (bool): store all state in local process if falsy,
-                else write state to Redis service
-            redis_port (int): redis server port number.
+            ip_states (MutableMapping[MutableMapping[str, IPDesc]]): dependency
```

Comment:
Don't leak implementation details here - just say that this is the backing mutable mapping where IP descriptors are stored